### PR TITLE
small fix and add SCW link from email flow

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -82,16 +83,22 @@ const cardContainer = (
...
   return classnames(
     position('relative'),
     borderRadius('rounded-2xl'),
     backgroundColor('bg-primary-dark'),
     cardColor(shadow ? color : undefined),
-    padding(small ? 'p-small' : 'p-6'),
+    padding({
+      'px-small': paddingType === 'small',
+      'py-small': paddingType === 'small',
+      'px-5': paddingType !== 'small',
```
Comment (T-Damer) [inline src/components/Card.tsx:98]: `!== 'small'` or `'normal'`?

File: src/components/Card.tsx [hunk 2]
```diff
@@ -102,7 +109,7 @@ const cardContainer = (
       onlyWrap ? undefined : 'sm:max-h-card',
       onlyWrap ? undefined : 'max-h-mobile-card'
     ),
-    space(nospace ? undefined : 'space-y-4'),
+    space(nospace ? undefined : 'space-y-6'),
```
Comment (T-Damer) [inline src/components/Card.tsx:112]: Let's use `space({'space-y-6': !nospace})`

File: src/components/landing/InitialCard.tsx [hunk 3]
```diff
@@ -41,6 +38,9 @@ export default function () {
           prove ownership of an NFT without it tracing back to you.
         </BodyText>
         <GetStartedButton />
+        <div className={alphaSealWrapper}>
```
Comment (T-Damer) [inline src/components/landing/InitialCard.tsx:41]: Why do we need to move this?

**Review by T-Damer — APPROVED** [review]
Looks cool, let's clean a few things :)

File: src/components/landing/InitialCard.tsx (hunk 3 above)
Comment (MixailE) [reply src/components/landing/InitialCard.tsx:41]: If this is positioned at the beginning, it adds padding even though the element is positioned absolutely

File: src/components/Card.tsx (hunk 1 above)
Comment (MixailE) [reply src/components/Card.tsx:98]: `paddingType: small` => `px-small py-small`
`paddingType: undefined` => `px-5 py-8`
`paddingType: normal` => `px-5 py-5`

**Pull request merged**